### PR TITLE
added testcontainer for RabbitMQ

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Added a testcontainer for the message broker RabbitMQ.
It returns ready-to-use config params for the `pika` client library.
This library was added to the dependencies.

### CODE REVIEW

**Comment by KerstenBreuer** [comment]
Hi @tillahoffmann and @SergeyPirogov,

could someone approve the workflow execution?

In general, I was wondering whether the approvals for workflow executions are really necessary. When looking at the other PRs, I see that most contributions already stall at this early step and don't even make it to the review.

**Comment by tillahoffmann** [comment]
Hi @KerstenBreuer, thanks for the contribution. Unfortunately, the first time approval is a security feature that can't be turned off ([for the time being](https://github.community/t/how-to-auto-approve-workflow-runs-for-first-time-contributors/176436)). I think there's a few linting errors, but then we can have a look.

**Comment by naomielst** [comment]
Hi, I [fixed the linting errors](https://github.com/KerstenBreuer/testcontainers-python/pull/1) in order to move this along. I am looking forward to using this new feature.

**Comment by thedrow** [comment]
@KerstenBreuer Could you please merge KerstenBreuer/testcontainers-python#1 to your branch so that the linter erros would be fixed. Thanks.

File: README.rst [hunk 1]
```diff
@@ -75,4 +76,4 @@ We recommend you use a `virtual environment <https://virtualenv.pypa.io/en/stabl
 Adding requirements
 ^^^^^^^^^^^^^^^^^^^
 
-We use :code:`pip-tools` to resolve and manage dependencies. If you need to add a dependency to testcontainers or one of the extras, run :code:`pip install pip-tools` followed by :code:`make requirements` to update the requirements files.
+We use :code:`pip-tools` to resolve and manage dependencies. If you need to add a dependency to testcontainers or one of the extras, modify the :code:`setup.py` as well as the :code:`requirements.ini` accordingly and then run :code:`pip install pip-tools` followed by :code:`make requirements` to update the requirements files.
```
[the remark this answers is not in the document]
Comment (KerstenBreuer) [reply README.rst]: Good spot, thanks

**Comment by KerstenBreuer** [comment]
Sorry for the delay on this PR. Could somebody approve the CI execution, again? @tillahoffmann or @yakimka?

**Comment by yakimka** [comment]
> Could somebody approve the CI execution, again? @tillahoffmann or @yakimka?

Sorry, I can't; I am not a collaborator :)

**Comment by KerstenBreuer** [comment]
Hi @tillahoffmann or @SergeyPirogov, could you please approve the workflow execution?

**Comment by KerstenBreuer** [comment]
Great, thanks a lot @tillahoffmann or @SergeyPirogov (whoever approved the workflow execution).

All the checks are passing. This PR is ready for review.

**Comment by KerstenBreuer** [comment]
Thanks @SergeyPirogov

**Comment by KerstenBreuer** [comment]
Hi @tillahoffmann, @SergeyPirogov,

do you know why the tests on the current main are failing: https://github.com/testcontainers/testcontainers-python/commit/6fb1a686578aa3380e8b08f72b16707e443331c4
(There is a problem with a test for the webdriver_container for python 3.8. This is something we didn't touch in this PR.)

Right before the merge, the tests passed on this PR.

Is there a difference between the checks that run on a PR vs. the checks that run on the main branch?

Do you know what went wrong?

**Comment by tillahoffmann** [comment]
I'm afraid this is likely down to failing tests. Hopefully that's something we can tackle together as part of #163.

**Pull request merged**